app/helpers/application_helper.rb

```diff
@@ -36,14 +38,13 @@ def hearings_sorter_link(prosecution_case, column, title = nil)
   end
 
   def app_environment
-    "app-environment-#{ENV.fetch('ENV', 'local')}"
+    "app-environment-#{ENV.fetch('ENV', nil) || 'local'}"
```

Comment:
What difference does this change make?

Reply:
This was a Rubocop automatic correction: 
![image](https://user-images.githubusercontent.com/26544538/167429715-dc0cf4b7-7486-4eb9-8399-ad2af9e65505.png)
This is the reason, quoted from [here](https://docs.rubocop.org/rubocop/cops_style.html#references-37): 

> This cop suggests ENV.fetch for the replacement of ENV[]. ENV[] silently fails and returns nil when the environment variable is unset, which may cause unexpected behaviors when the developer forgets to set it. On the other hand, ENV.fetch raises KeyError or returns the explicitly specified default value.

Reply:
oh you're right - I must have been running against old code as you said, reverting back to:
`app-environment-#{ENV.fetch('ENV', 'local')}`